sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedIterableBase.java

```diff
@@ -80,7 +80,7 @@ public PagedIterableBase(PagedFluxBase<T, P> pagedFluxBase) {
      *
      * @param provider the Page Retrieval Provider
      */
-    PagedIterableBase(Supplier<PageRetrieverSync<String, P>> provider) {
```

Comment:
is this intended?

Reply:
yes, it is needed to unblock the custom Paged Iterable implementations in Search SDK.